core/maven-example/pom.xml

```diff
@@ -24,7 +31,7 @@
     <dependency>
       <groupId>jakarta.servlet</groupId>
       <artifactId>jakarta.servlet-api</artifactId>
-      <version>5.0.0</version>
+      <version>6.1.0</version>
```

Comment:
This is Jakarta EE 11, while this repository is currently on Java EE 9.

Reply:
Agreed, this should have been left at 5.0.0 (EE 9). When the repository is upgraded to e.g. EE 10 in the future (cf. #617), we can upgrade it then.